docs/_static/custom.css

```diff
@@ -81,6 +81,14 @@ div.admonition, div.note {  /* Ignore CSSLintBear */
     transition: box-shadow .25s;
 }
 
+div.caution {  /* Ignore CSSLintBear */
+    background-color: #b44c0e;
+}
+
+div.important {  /* Ignore CSSLintBear */
+    background-color: #783238;
+}
```

Comment:
Isn't this maybe a bit too dark?

Btw I would like to see a screenshot how it looks like when it's rendered :)

Reply:
We should netlify the docs in this repo.
I had a WIP for that, but it was blocked by https://github.com/coala/devops/issues/90

Reply:
I have added a screenshot in the comments anyway. Is that okay or any changes required ? @jayvdb